### ISSUE TITLE
composer update fails

### ISSUE THREAD

**Issue by mrshowerman** [simplesamlphp/composer-module-installer#18]
Since v1.1.10, I get this error on composer update:

```
  - Installing simplesamlphp/composer-module-installer (v1.1.10): Extracting archive
Plugin initialization failed (Plugin simplesamlphp/composer-module-installer could not be initialized, class not found: SimpleSAML\Composer\ModuleInstallerPlugin), uninstalling plugin
  - Removing simplesamlphp/composer-module-installer (v1.1.10)
```

The cause might be the difference between the file path (src/SimpleSamlPhp/Composer/ModuleInstallerPlugin.php) and the namespace (\SimpleSAML\Composer\ModuleInstallerPlugin).

**Comment by tvdijen** [comment]
Looking at it right now!

**Comment by tvdijen** [comment]
It got it fixed in v1.1.12! Sorry for the inconvenience all!

**Comment by mrshowerman** [comment]
That was quick, thanks a lot.
